### DISCUSSION TITLE
how to use two differente package version in one project?

### DISCUSSION THREAD

**Discussion by melonges** [WalletConnect/walletconnect-monorepo#2777] [Developer Support]
I use nx. Nx has one package.json and node_modules.
I have problem with two differente package version
<img width="665" alt="image" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/70666395/048678e7-eb76-4f2b-875b-4c84fc3b9196">
<img width="1643" alt="image" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/70666395/47fe11eb-d026-41ba-a7aa-76126601c2aa">

**Comment by melonges** [comment]
wagmi is problem 
<img width="459" alt="image" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/70666395/20314106-c076-4509-a759-8e45404c4e54">

**Comment by finessevanes** [comment]
@melonges are you still facing issues?

**Reply by melonges** [reply]
yes, the problem remains. I think its difference wagmi version

**Comment by glitch-txs** [comment]
Is that the only error you get?
Have you tried adding `//@ts-ignore`?

**Comment by melonges** [comment]
I start rewrite projects to single library version
